packages/module/patternfly-docs/content/extensions/component-groups/examples/ExpandableText/ExpandableText.md

```diff
@@ -0,0 +1,23 @@
+---
+section: extensions
+subsection: Component groups
+id: ExpandableText
+source: react
+propComponents: ['ExpandableText']
+---
+
+import ExpandableText from "@patternfly/react-component-groups/dist/dynamic/ExpandableText";
+
+## Component usage
+
+### ExpandableText component
```

Comment:
We should probably add some description similar tho the rest of the components

Reply:
Done.